classes/local/rule_checker.php

```diff
@@ -145,14 +214,23 @@ public function is_registration_allowed(): bool {
         return true;
     }
 
+    /**
+     * Return checks' resulting messages.
```

Comment:
Can't work out if the apostrophe is correct here, might be better written as "Return the resulting messages from each check"?